src/onboarding2/Templates/TemplateDetails.js

```diff
@@ -0,0 +1,166 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Button,
+  Field,
+  GU,
+  Link,
+  Modal,
+  textStyle,
+  useViewport,
+  useTheme,
+} from '@aragon/ui'
+import { simpleUrl } from '../../url-utils'
+import AppIcon from '../../components/AppIcon/AppIcon'
+
+function TemplateDetails({ template, visible, onUse, onClose }) {
+  const theme = useTheme()
+  const { above, below, width } = useViewport()
+
+  const handleUseClick = useCallback(() => {
+    onUse(template.id)
+  }, [onUse, template])
+
+  const modalWidth = useCallback(() => {
+    if (above('large')) {
+      return 130 * GU
+    }
+    if (above('medium')) {
+      return 80 * GU
+    }
+    return width - 4 * GU
+  }, [above, width])
+
+  if (template === null) {
+    return null
+  }
+
+  let padding = 2 * GU
+  if (above('medium')) padding = 5 * GU
```

Comment:
`layoutName === 'medium'` && `layoutName === 'large'` ?

Reply:
There is no `Layout` context here.

Reply:
Can we add it? Why wouldn't it work?